exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -1605,6 +1621,7 @@ func (b *Bitfinex) GenerateDefaultSubscriptions() ([]stream.ChannelSubscription,
 					formattedPair = wsPairFormat.Format(enabledPairs[k])
 				}
 
+				// TODO - This should not be here in DefSubs
```

Comment:
oooh! I leaked this thought.
It's a good one though.
I don't think that DefSubs should be responsible for setting immutable Params.
We should move this to Subscribe.
Perhaps hit that TODO on 1626 at the same time.